lib/stripe/changeset.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Stripe.Changeset do
+  alias Stripe.Util
+
+  def cast(params, schema) do
+    keys = Map.keys(schema)
+    params = Util.atomize_keys(params)
+
+    Enum.reduce(keys, %{}, fn key, acc ->
+      case Map.get(params, key) do
+        value when is_nil(value) ->
+          acc # remove nil values
```

Comment:
This is introducing a bug since some values in Stripe's API can be unset with `null` values. Going to make `cast/3` instead which accepts an optional list of params that are nullable.